src/backend/setup.cfg

```diff
@@ -26,27 +26,30 @@ requires-python = >=3.6
 
 [options]
 install_requires =
```

Comment:
Can you sort updates alphabetically in your commit message?

Reply:
done